### ISSUE TITLE
Authorization method not supported

### ISSUE THREAD

**Issue by tylerlong** [grokify/ringcentral-oauth-demos#9]
I was able to get the access_key, but the key was "useless". When I made a get request to `/restapi/v1.0/dictionary/timezone/12`, I got the following exception:

```
An exception of type 'RingCentral.Http.ApiException' occurred in RingCentral.dll but was not handled in user code

Additional information: Authorization method not supported
```

![image](https://cloud.githubusercontent.com/assets/733544/16482454/680b7072-3ee2-11e6-9b52-5c5805cbccd1.png)

I am not sure it is a bug of the C# SDK or an issue from server side. Let me investigate.

**Update**

Tested `/restapi/v1.0/account/~/extension/~`, same issue

**Update**

Unable to reproduce the issue with JS SDK!  This seems like a C# only issue.

**Comment by tylerlong** [comment]
I confirm it is issues rooted in C# SDK. 

I have fixed it: https://github.com/ringcentral/ringcentral-csharp/pull/46/commits/ba30a7f5aa484dc4e6d8eb1ce1a9527ec6d0d701  I am going to release a new NuGet package.

Feel free to close this issue.

**Comment by tylerlong** [comment]
I have released 1.0.1 version to NuGet.

Thanks to @LingForCC . He found the issue and reported to me.
